corehq/ex-submodules/casexml/apps/phone/models.py

```diff
@@ -1307,12 +1297,6 @@ def get_properly_wrapped_sync_log(doc_id):
     except ValidationError:
         # this occurs if doc_id is not a valid UUID
         synclog = None
-    if not synclog:
-        try:
-            # try to lookup in couch
-            return properly_wrap_sync_log(SyncLog.get_db().get(doc_id))
-        except ResourceNotFound:
-            pass
         raise MissingSyncLog("A SyncLogSQL object with this synclog_id ({})is not found".format(
```

Comment:
this will only get raised if there is a `ValidationError` whereas it is currently raised if there is no synclog found

Reply:
Ha, right. I will fix that.